kitsune/dashboards/jinja2/dashboards/localization.html

```diff
@@ -83,21 +83,8 @@ <h2>{{ _('Locale:') }} {{ current_locale_name }}</h2>
                                      (overview_rows['all'], True),
                                      (overview_rows['templates'], True))) }}
 
-      <div id="stats-tabs" data-ui-type="tabbed-view">
```

Comment:
We can replace this `div` with `<nav class="tabs">` from protocol and apply the proper classes (`tabs--list, tabs--item, tabs--link`) to the ul, li, a elements. This should probably give us a proper tabbed menu which will required small changes in css and js.

Reply:
@akatsoulas I created #5786 as an alternative to consider.